api/src/pcapi/routes/backoffice_v3/templates/accounts/get/general.html

```diff
@@ -1,12 +1,14 @@
 {% from "components/forms.html" import build_form_fields_group with context %}
 {% from "components/generic_modal.html" import build_modal_form with context %}
+{% import "components/clipboard.html" as clipboard %}
 <div class="row row-cols-1 g-4 py-3">
   <div class="col">
     <div class="card">
       <div class="card-body">
         <div class="d-flex flex-row justify-content-start">
           <h5 class="card-title">
-            {{ user.firstName | empty_string_if_null }} {{ user.lastName | empty_string_if_null | upper }}
+            {{ username }}
```

Comment:
Pour éviter de passer la mise en forme à `render_template`, tu dois pouvoir stocker dans une variable : 
```suggestion
            {% set username = user.firstName | empty_string_if_null + ' ' + user.lastName | empty_string_if_null | upper %}
            {{ username }}
```